src/ripple/protocol/jss.h

```diff
@@ -317,6 +317,7 @@ JSS(fee_level);             // out: AccountInfo
 JSS(fee_mult_max);          // in: TransactionSign
 JSS(fee_ref);               // out: NetworkOPs
 JSS(fetch_pack);            // out: NetworkOPs
+JSS(FIELDS);                // out: RPC server_definitions
```

Comment:
This is the format that is being copied: https://github.com/XRPLF/xrpl.js/blob/main/packages/ripple-binary-codec/src/enums/definitions.json

Reply:
okay, I'll go through that file.
What is the rationale for the `CAPITALIZATION_EXCEPTIONS` dictionary? Are those terms being used in another context? https://github.com/RichardAH/xrpl-codec-gen/blob/9157e62e7d585d9dd1bb17eb3f74f2a209fddcd0/gen.py#L4

Reply:
That's because it's parsing the enums - e.g. `XCHAIN_BRIDGE` - which needs to be turned into `XChainBridge`. If you naively capitalize when converting from snake_case to CamelCase you'll get `XchainBridge`. Hence the exceptions.

Reply:
Please add a short comment to every field that does not follow the naming convention and explain that it was done intentionally. (this, and things like `isSerialized`, `isSigningField`, `isVLEncoded`, `LEDGER_ENTRY_TYPES`

Reply:
Done in https://github.com/XRPLF/rippled/pull/4703/commits/ebe4258ac26d67cb544adfd7c7d693a3d72ce08e